Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /app
 COPY package.json ./
 
 RUN yarn config set cache-folder /app/.yarn_cache && \
-    yarn install
+    yarn install --ignore-optional
```

Comment:
¿No sería más conveniente [que no modifique el yarn.lock](https://stackoverflow.com/questions/58482655/what-is-the-closest-to-npm-ci-in-yarn)?

Reply:
Resuelto en https://github.com/carlotiii30/organizacionSemanal/pull/99/commits/2906e8017d55a0c6dc6e313eb0e44bb312ce431b. Gracias por el consejo.